html_sanitizer.rst

```diff
@@ -793,7 +792,7 @@ URLs of ``<a>`` elements:
                     // specifies the allowed hosts, the attribute will be dropped if the
                     // URL contains a different host. Subdomains are allowed: e.g. the following
                     // config would also allow 'www.symfony.com', 'live.symfony.com', etc.
-                    ->allowedLinkHost('symfony.com')
+                    ->allowedLinkHosts('symfony.com')
```

Comment:
Is this really a string, I would expect and array here 🤔

Reply:
In the config class created by Symfony I see this:

```php
    /**
     * Allows only a given list of hosts to be used in links href attributes.
     * @default null
     * @param ParamConfigurator|mixed $value
     *
     * @return $this
     */
    public function allowedLinkHosts(mixed $value = NULL): static
    {
        $this->_usedProperties['allowedLinkHosts'] = true;
        $this->allowedLinkHosts = $value;

        return $this;
    }
```

Reply:
You are right. The CI even confirms this after merging symfony-tools/symfony-application#19.